ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -101,6 +102,7 @@ public String getStatementText() {
     return windowType;
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "fields is ImmutableList")
   public List<FieldInfo> getFields() {
```

Comment:
This goes for a lot of other files as well, but should we change the internal representation to `ImmutableList`?